cabal.project

```diff
@@ -178,37 +178,9 @@ package cardano-wallet-test-utils
 package strict-non-empty-containers
   tests: True
 
--- Disable all other tests. This is what they do in the cardano-node
--- cabal project.
--- The global tests: False flag doesn't seem to work though.
+-- Now disable all other tests with a global flag.
```

Comment:
Does this actually work? I've always had trouble with cabal.project's and global flags. 

I've tried things like

```
package *:
  tests: False
```

In the past but haven't had it work.

Reply:
Always confuses me.

Reply:
It seems to work for me locally, and the CI builds also work (whereas before they were failing due to difficult test suite dependencies). I think key part is using Cabal 3.4.0.0.